PluginFramework/src/LoadStrategies/PluginFolder.cs

```diff
@@ -0,0 +1,65 @@
+﻿using PluginFramework.src.Interfaces.Manager;
+using PluginFramework.src.Interfaces.PluginTypes;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace PluginFramework.src.LoadStrategies
+{
+    /// <summary>
+    /// Load strategy to load all plugins in a folder
+    /// </summary>
+    public class PluginFolder : IPluginLoadStrategy
+    {
+        /// <summary>
+        /// The folder to search the plugins ins
+        /// </summary>
+        private readonly string folder;
+
+        /// <summary>
+        /// Create a new instance of this load strategy
+        /// </summary>
+        /// <param name="folder">The folder to search for plugins</param>
+        public PluginFolder(string folder)
+        {
+            this.folder = folder;
+        }
+
+        /// <inheritdoc/>
+        public List<T> LoadPlugins<T>() where T : IPluginOverhead
+        {
+            List<T> returnList = new List<T>();
+            if (!Directory.Exists(folder))
+            {
+                return returnList;
+            }
+            foreach (string file in Directory.GetFiles(folder))
+            {
+                FileInfo fileInfo = new FileInfo(file);
+                if (fileInfo.Extension != ".dll")
+                {
+                    continue;
+                }
+
+                Assembly assembly = Assembly.LoadFrom(file);
+                try
+                {
+                    foreach (Type type in assembly.GetTypes())
+                    {
+                        if (type.GetInterfaces().Contains(typeof(T)))
+                        {
+                            T formatter = (T)Activator.CreateInstance(type);
+                            returnList.Add(formatter);
+                        }
+                    }
+                }
+                catch (Exception)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5494671)